agentops/instrumentation/mem0/common.py

```diff
@@ -0,0 +1,377 @@
+"""Common utilities and base wrapper functions for Mem0 instrumentation."""
+
+from typing import Dict, Any
+from opentelemetry import context as context_api
+from opentelemetry.trace import SpanKind, Status, StatusCode
+
+from agentops.instrumentation.common.attributes import AttributeMap
+from agentops.semconv import SpanAttributes, LLMRequestTypeValues
+
+
+def get_common_mem0_attributes() -> AttributeMap:
```

Comment:
Since we're inside the `mem0` instrumentation, functions don't need to include the name. 

`get_common_attributes` is fine here.